db/seeds.d/75-job_templates.rb

```diff
@@ -3,7 +3,12 @@
     JobTemplate.without_auditing do
       Dir[File.join("#{Katello::Engine.root}/app/views/foreman/job_templates/**/*.erb")].each do |template|
         sync = !Rails.env.test? && Setting[:remote_execution_sync_templates]
-        JobTemplate.import!(File.read(template), :default => true, :locked => true, :update => sync)
+        # import! was renamed to import_raw! around 1.3.1
+        if JobTemplate.respond_to?('import_raw!')
+          JobTemplate.import_raw!(File.read(template), :default => true, :locked => true, :update => sync)
+        else
+          JobTemplate.import!(File.read(template), :default => true, :locked => true, :update => sync)
```

Comment:
Is there a "public interface" method that could be used instead of calling this internal method that can change?